bundles/AdminBundle/Controller/Admin/SettingsController.php

```diff
@@ -236,11 +236,14 @@ public function getPredefinedMetadataAction(Request $request)
     {
         $type = $request->get('type');
         $subType = $request->get('subType');
+        $group = $request->get('group');
         $list = Metadata\Predefined\Listing::getByTargetType($type, [$subType]);
         $result = [];
         foreach ($list as $item) {
-            $item->expand();
-            $result[] = $item->getObjectVars();
+            if ((string)$group === $item->getGroup()) {
+                $item->expand();
```

Comment:
this doesn't work anymore since the properties are protected now (since 10).
please add $item = $item->getObjectVars() after $item->expand()

Reply:
Fixed. (I actually developed this for Pimcore 6 and just copied the changes)